yt_dlp/extractor/rinsefm.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime, timezone
+
+from .common import InfoExtractor
+
+
+class RinseFMIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rinse\.fm/episodes/(?P<id>.+)'
```

Comment:
We don't want to match query/fragment/etc
```suggestion
    _VALID_URL = r'https?://(?:www\.)?rinse\.fm/episodes/(?P<id>[^/?#]+)'
```